grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -3332,7 +3332,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Number of mutations (ALTER DELETE/ALTER UPDATE)",
+      "description": "Number of active mutations (ALTER DELETE/ALTER UPDATE) and how much parts should be mutate",
```

Comment:
Let's make it shorter:

"Number of active mutations (ALTER DELETE/ALTER UPDATE) and parts to mutate"